lookout/style/format/benchmarks/evaluate_smoke.py

```diff
@@ -300,7 +299,7 @@ def evaluate_smoke_entry(inputpath: str, reportdir: str, database: str, bblfsh:
     start_time = time.time()
     report_filename = os.path.join(reportdir, "report.csv")
     log = logging.getLogger("evaluate_smoke")
-    port = server.find_port()
+    lookout = LookoutSDK()
```

Comment:
```suggestion
    lookout_sdk = LookoutSDK()
```

(we've got `lookout` package)